examples/python_sdk/cancel_create_experiment.py

```diff
@@ -0,0 +1,43 @@
+from determined.common import yaml
+from determined.common.experimental.determined import Determined
+
+
+def cancel_experiment(D: Determined) -> None:
+    exp = D.get_experiment(experiment_id=1)
```

Comment:
in this example, let's parameterize experiment_id, because using `1` is not going to work most of the time.